Library/DiscUtils.Core/Internal/Utilities.cs

```diff
@@ -1023,7 +996,7 @@ public static bool Is8Dot3(string name)
 
         public static bool Is8Dot3Char(char ch)
         {
-            return (ch >= 'A' && ch <= 'Z') || (ch >= '0' && ch <= '9') || "_^$~!#%�-{}()@'`&".IndexOf(ch) != -1;
+            return (ch >= 'A' && ch <= 'Z') || (ch >= '0' && ch <= '9') || "_^$~!#%£-{}()@'`&".IndexOf(ch) != -1;
```

Comment:
Yes, it's UTF-8 now (with BOM) and the £ is uniquely defined in UTF-8. The reason it was mangled in the original e-mail is because the £ sign does not appear in ASCII, so it was encoded using one of the Windows-specific codepages. Should be all good now.